learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 2 Learning Outcomes
+
+## HTTP
+
+### Asynchronicity
+
+- [ ] **We can write code that executes asynchronously**
+- [ ] We can use callbacks to make our code non blocking
+
+### Fetch
+
+- [ ] **We can use the fetch method to make API requests and receive responses**
+- [ ] We can configure the options argument of the fetch method to make GET and POST requests
+- [ ] We can use an API key to bypass rate limit restrictions on certain APIs
+
+- [ ] **We can use the map and filter methods to display different data based on user input**
+- [ ] We can return filtered items from an array that match specified criteria
+- [ ] We can map over an array to create a new array containing new values
+
+- [ ] **We can manipulate the DOM using JavaScript**
+- [ ] We can use different selector methods to select DOM nodes
+- [ ] We can add and remove DOM nodes to change the appearance of our website
+- [ ] We can toggle the classes applied to DOM nodes to change their properties
+
+## Design
+
+- [ ] **We can use consistent layout and spacing**
+- [ ] We can implement readability margins to ensure a comfortable reading experience on different devices
+- [ ] We can follow a spacing guideline to give our app a consistent feel
+- [ ] We can strike a good balance between active space and white space in our app
+
+## Developer Toolkit LOs
+
+- [ ] **We can debug client side JS in our web browser**
+- [ ] We can inspect the DOM in our browser to see which elements are there
+- [ ] We can view the console in our browser to help us debug our JavaScript
+- [ ] We can use console.log() to help us debug our code
+- [ ] We can setup Prettier so that our code is more readable
```

Comment:
Bit pedantic sorry, but "setup" is a noun. "set up" is the verb. Also I'd say the most important Prettier benefit to highlight is consistency within a team

```suggestion
- [ ] We can set up Prettier so that our formatting is consistent across the team
```

Reply:
That's a great point, I agree that's the most important benefit of using Prettier